invenio.cfg

```diff
@@ -410,24 +410,29 @@ RDM_FILES_DEFAULT_MAX_FILE_SIZE = 50 * 10 ** 9  # 50GB
 
 JOBS_ADMINISTRATION_ENABLED = True
 
-RDM_RECORDS_IDENTIFIERS_SCHEMES = {**RDM_RECORDS_IDENTIFIERS_SCHEMES,
-                                   **{"cds_ref": {"label": _("CDS Reference"),
-                                                  "validator": always_valid,
-                                                  "datacite": "CDS"},
-                                      "aleph": {"label": _("Aleph number"),
-                                                "validator": schemes.is_aleph,
-                                                "datacite": "ALEPH"},
+RDM_RECORDS_IDENTIFIERS_SCHEMES ={
+                                "cds_ref": {"label": _("CDS Reference"),
```

Comment:
@sakshamarora1 was maybe working on renaming this to `cdsrn`? Or am I mistaken?

Reply:
Yes, so either one of the PR will need to be updated again depending on which goes first.
#569

Reply:
can we do both ? @0einstein0 can you please change this part to cdsrn ?